econml/metalearners.py

```diff
@@ -185,7 +199,8 @@ def marginal_effect(self, X):
         τ_hat : array-like, shape (m, )
             Matrix of heterogeneous treatment effects for each sample.
         """
-        return self.effect(X)
+        T0 = self._one_hot_encoder.categories_[0][0]
+        return self.effect(X, T0=T0, T1=T1)
```

Comment:
Again this would probably be better defined by the LinearCateEstimator's const_marginal_effect, although it's not obvious that the idea of a marginal effect really makes sense for the SLearner.  Maybe something to dig into during our SDK sync.